lib/views/create_project_form.dart

```diff
@@ -86,6 +94,30 @@ class _CreateProjectFormState extends State<CreateProjectForm> {
     );
   }
 
+  LenraButton cancelButton(bool hasApp) {
+    if (hasApp) {
+      return LenraButton(
+        text: "Cancel",
+        onPressed: () {
+          CommonNavigator.go(context, BackofficeNavigator.selectProject);
```

Comment:
Should do a back for if you come from another page.

Reply:
I tried to do a CommonNavigator.pop before but it does not seem to work because we do not properly handle the navigation across views, we might need to do this on another PR.